### DISCUSSION TITLE
Lazy - unknown problem after plugin update today

### DISCUSSION THREAD

**Discussion by o0Noise0o** [LazyVim/LazyVim#4210] [Q&A]
Hello all,

i was enjoying Lazyvim since a few days but after Lazy updated everything i am getting an error where i dont even know where to begin looking. Can u please advice again?

![rgr](https://github.com/user-attachments/assets/a1df977c-3f6a-4732-8214-827173bff914)

**Comment by jellis206** [comment]
if you can find where all your plugins are downloaded I usually start by deleting all of those if something like this happens. Sometimes a fresh install of all the plugins will clear things up

**Comment by 0x2E** [comment]
Reinstall plugins: https://github.com/LazyVim/LazyVim/discussions/3007#discussioncomment-9163986
